src/app/features/collection/components/cards-list-skeleton/cards-list-skeleton.component.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { classWithProviders } from '@ngx-unit-test/inject-mocks';
```

Comment:
this file looks like can be removed, since it does not contains some unit tests that test logic